prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md

```diff
@@ -0,0 +1,39 @@
+# prep-html-skeleton-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What tags are necessary for a complete HTML Skeleton?
+
```

Comment:
Hey Bryan, make sure to answer the questions here so you can save them and have them to review!